### DISCUSSION TITLE
Can milkdown render html tags ?

### DISCUSSION THREAD

**Discussion by Sma11X** [Milkdown/milkdown#1037] [Q&A] [answered]
I want to use `<details>` tag for hiding some long text but can't find an example about it.
Is there any ways to implement feature like that or should write a new plugin ?

**Comment by roma-marshall** [comment, marked as answer]
hello @Sma11X, check it out: https://github.com/Milkdown/milkdown/issues/126

**Reply by Sma11X** [reply]
I've tried to write a custom plugin already.
Thanks for replying!

**Reply by JustinyAhin** [reply]
@Sma11X do you mind sharing your custom plugin implementation please?
